dbusmock/templates/bluez5.py

```diff
@@ -111,6 +111,9 @@ def AddAdapter(self, device_name, system_name):
     Returns the new object path.
     '''
     path = '/org/bluez/' + device_name
+    address_start = int(device_name[-1])
+    address = f"{address_start:02d}:{address_start+1:02d}:{address_start+2:02d}:" + \
+        f"{address_start+3:02d}:{address_start+4:02d}:{address_start+5:02d}"
```

Comment:
This would be cleaner in mod 256 (aka `& 0xFF`), but let's hope that device names are ASCII :grin:

Reply:
I really didn't bother with any argument checking, the code above doesn't even check for a valid device_name that could also be a valid D-Bus path object component. Let me know if you want me to add more checks here.